content/guides/user-guides/csi/ceph csi/_index.md

```diff
@@ -0,0 +1,396 @@
+---
+title: CEPH CSI User Guide
+description: "A user guide for integrating SODA with CEPH CSI Driver via unified Plugin and hotpot project"
+weight: 30
+disableToc: false
+tags: ["user guide", "csi", "kubernetes csi", "plug-n-play", "unified plugin"]
+---
+
+# SODA Integration with Vendor CSI Drivers via SODA CSI Plug-N-Play
+
+SODA CSI Plug-N-Play is a mechanism by which users can use the heterogeneous storage vendors in a unified way, Users
+need to define the requirements while creating SODA Profile, Users need to use this profile ID while creating the PVC's
+in the Kubernetes, the soda-csi-provisioner will automatically select the vendor csi-driver and help in provisioning the
+Volumes for Pods.
+
+## Prerequisite
+
+- An installation of Kubernetes (V1.17+)
+- SODA installation (you can refer to quick start guide
+  over [here](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/)), you need to change the
+  ports of etcd in osdsdb.yaml while installing SODA on an existing K8s environment as the ports cause conflict with k8s
+  etcd.
+
+## Selecting the vendor CSI driver
+
+You can select the CSI driver supported by SODA Plug-N-Play
+from [here](https://docs.sodafoundation.io/guides/user-guides/nbp/csi-pnp/) Once selected follow the instruction given
+in the document to deploy the CSI driver in K8s.
+
+##### For this example we are selecting the [CEPH-RBD CSI Driver](https://github.com/ceph/ceph-csi).
+
+Following are some additional steps we need to perform for this plugin
```

Comment:
Following are the pre-requisites for Ceph CSI Driver

Reply:
changes done.